### PR TITLE
Network 2018-02-01

### PR DESCRIPTION
**Pull request by murilogr**
## Description

Automatically generated based on latest REST API specs.

[REST API](https://github.com/Azure/azure-rest-api-specs/pull/2642)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [?] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@murilogr Please link the related rest spec PR

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please run
`msbuild build.proj /t:build /p:Scope=SDKs\Network` and commit any artifacts
Please create a `generate.ps1` file similar to [this ](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>17.0.0-preview</Version>
+		<Version>18.0.0-preview</Version>
 		<PackageTags>Microsoft Azure Network management;Network;Network management;</PackageTags>    
 		<PackageReleaseNotes>
 		<![CDATA[
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Please update the package release notes for this version

Comment (murilogr) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Fixed

**Comment by murilogr** [comment]
Link for the REST API merge: [REST API](https://github.com/Azure/azure-rest-api-specs/pull/2642)

File: src/SDKs/Network/Management.Network/Generated/SecurityRulesOperations.cs [hunk 2]
```diff
@@ -126,7 +126,7 @@ internal SecurityRulesOperations(NetworkManagementClient client)
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "this.Client.SubscriptionId");
             }
-            string apiVersion = "2018-01-01";
+            string apiVersion = "2018-02-01";
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Generated/SecurityRulesOperations.cs]: @murilogr can you confirm if this change was expected.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please run `msbuild build.proj /t:build /p:Scope=SDKs\Network`
This should modify the .props file, please commit these changes
Looks good other than that

File: src/SDKs/Network/Management.Network/Generated/SecurityRulesOperations.cs (hunk 2 above)
Comment (murilogr) [reply src/SDKs/Network/Management.Network/Generated/SecurityRulesOperations.cs]: It's expected. If you look at the diff from psSdkJson6 to this branch, we're just changing the version.

For some reason, my first generation used Client.ApiVersion, but the second one reverted to what it was, with "apiVersion".

**Comment by murilogr** [comment]
@dsgouda I updated props file generated by the build

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**